src/main/kotlin/com/coder_rangers/mobius_api/models/Category.kt

```diff
@@ -0,0 +1,34 @@
+package com.coder_rangers.mobius_api.models
+
+import com.coder_rangers.mobius_api.enums.CategoryType
+import javax.persistence.CascadeType.ALL
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.EnumType.STRING
+import javax.persistence.Enumerated
+import javax.persistence.Id
+import javax.persistence.OneToMany
+import javax.persistence.Table
+import javax.validation.constraints.NotBlank
+import javax.validation.constraints.PositiveOrZero
+
+@Entity
+@Table(name = "categories")
+class Category(
+    @Id
+    @field:PositiveOrZero
+    @Column(unique = true, nullable = false, updatable = false)
+    val id: Long,
+
+    @Enumerated(STRING)
+    @field:NotBlank
+    @Column(nullable = false)
+    val type: CategoryType,
+
+    @field:NotBlank
+    @Column(nullable = false, length = 255)
+    val description: String,
+
+    @OneToMany(cascade = [ALL])
```

Comment:
A esta annotation le falta algo para mapear bien contra una columna y que no se genere una tabla intermedia.

```suggestion
    @OneToMany(mappedBy = "category", cascade = [ALL])
```

Está en el link que te pasé de baeldung sobre las relaciones en Spring JPA.